packages/features/users/components/AdminPasswordBanner.tsx

```diff
@@ -3,17 +3,19 @@ import Link from "next/link";
 
 import { useLocale } from "@calcom/lib/hooks/useLocale";
 import { TopBanner } from "@calcom/ui";
+import { trpc } from "@calcom/trpc/react";
 
 function AdminPasswordBanner() {
   const { t } = useLocale();
   const { data } = useSession();
+  const { data: user, isLoading } = trpc.viewer.me.useQuery();
 
   if (data?.user.role !== "INACTIVE_ADMIN") return null;
 
   return (
     <>
       <TopBanner
-        text={t("invalid_admin_password", { user: data.user.username })}
+       text={user?.twoFactorEnabled ? t("invalid_admin_password", { user: data.user.username }) : t("invalid_admin_password_and_no_2FA", { user: data.user.username }) }
```

Comment:
@Ninad3721 we displayed the message "You are admin but you do not have a password length of at least 15 characters or no 2FA yet" before but after this change if a user updates a password to 15 characters and 2FA disabled he would see the message 

"You are admin but you do not have a password length of at least 15 characters and no 2FA yet" which is not accurate